<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_principal</class>
 <widget class="QMainWindow" name="fm_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de gestão de cestas de café da manhã - FCMCestas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource_files.qrc">
    <normaloff>:/icons/images/icone.png</normaloff>:/icons/images/icone.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pesquisar:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pesquisar"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_pesquisar">
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_mostrar_todos">
        <property name="text">
         <string>Mostrar todos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget_cestas"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ver_cesta">
        <property name="text">
         <string>Ver cesta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_nova_cesta">
        <property name="text">
         <string>Nova cesta</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_versao">
        <property name="text">
         <string>Versão 1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_autenticacao">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource_files.qrc">
          <normaloff>:/icons/images/close.png</normaloff>:/icons/images/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_colaborador_logado">
        <property name="text">
         <string>Autentique-se!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_fechar">
        <property name="text">
         <string>Fechar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionColaboradores"/>
    <addaction name="actionProdutos"/>
    <addaction name="separator"/>
    <addaction name="actionSobre"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionProdutos">
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="actionColaboradores">
   <property name="text">
    <string>Colaboradores</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton_autenticacao</tabstop>
 </tabstops>
 <resources>
  <include location="resource_files.qrc"/>
 </resources>
 <connections/>
</ui>
